components/v-mapbox-components/v-mapbox-vector-layer.vue

```diff
@@ -20,6 +24,7 @@ export default {
   data() {
     return {
       map: null,
+      layerToZoomTo: '',
```

Comment:
null/''?

Reply:
This needs to be removed, not used anymore